### PR TITLE
Add dumb simple GHA

### PR DESCRIPTION
**Pull request by EvanPurkhiser**

### CODE REVIEW

**Comment by EvanPurkhiser** [comment]
@billyvg lmk if you think there's more checks I could add to this

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: build
+on: [push]
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```
Comment (billyvg) [inline .github/workflows/build.yml:9]: Doesn't matter in this repo, but prob better to use v2 (it doesn't checkout entire repo, but just HEAD commit by default)

**Review by billyvg — APPROVED** [review]
you could add a docker build?

File: .github/workflows/build.yml (hunk 1 above)
Comment (EvanPurkhiser) [reply .github/workflows/build.yml:9]: copy pasted this from one of my side projects, thanks

**Pull request merged**